docs/animations/AnimationSet.md

```diff
@@ -63,8 +91,17 @@ You can change the way how the animation interpolates between keyframes by defin
 | Quintic    | Create an animation that accelerates or decelerates using the formula f(t) = t5                                                                         | ![QuinticEase](https://docs.microsoft.com/en-us/dotnet/framework/wpf/graphics-multimedia/media/quinticease-graph.png)     |
 | Sine       | Creates an animation that accelerates or decelerates using a sine formula                                                                               | ![SineEase](https://docs.microsoft.com/en-us/dotnet/framework/wpf/graphics-multimedia/media/sineease-graph.png)           |
 
-***Note:** EasingType is used only when AnimationSet.UseComposition == false*  
-***Note:** Blur, Light and Saturation animation don't support easing*
+> [!IMPORTANT]
+EasingType is used only when AnimationSet.UseComposition == false
+
```

Comment:
Note #1761

Reply:
Sorry for the late response, I had exams this week.

For now, I will keep it like this. If #1761 didn't solve this issue then we can change it.

Reply:
A Note is just a skinned Quote Block, which can handle some formatting:

```
>```csharp
> void 
>```
```

This works the same here on GitHub, as it does in the MarkdownTextBlock.

>```csharp
> void 
>```